### DISCUSSION TITLE
hhh版本号感觉多出一个 v

### DISCUSSION THREAD

**Discussion by Cxpller** [MaLuns/hexo-theme-async#31] [General 大局为重]
![image](https://user-images.githubusercontent.com/76519622/197445935-90fcefb0-f194-4266-99c4-3631c6b6a360.png)

**Comment by Cxpller** [comment]
感觉是我自己的问题，但是找不出地方来

**Reply by Cxpller** [reply]
这个是页面的地址 https://cxpller.github.io/

**Comment by Cxpller** [comment]
# 页脚Footer
footer:
  copyrightYear: 2022
  live_time:
    enable: true
    prefix: footer.tips
    start_time: 10/08/2022 12:02:00

**Comment by MaLuns** [comment]
是个 bug , 发布 npm 包版号里多了个 v
